app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -1,6 +1,12 @@
-attributes :content_view_id, :content_view_name
-attributes :lifecycle_environment_id, :lifecycle_environment_name
-attributes :uuid
+child :content_view => :content_view do
+  attributes :id, :name
+end
+
+child :lifecycle_environment => :lifecycle_environment do
+  attributes :id, :name
+end
```

Comment:
The point of having the object method (as opposed to the flat method) is that it follows our conventions elsewhere in the katello APIs.  I would argue that if we adopt the flat method then we should replace it everywhere else.  Keeping the object method is inline with what we do everywhere else and will make it easier for consumers of our API to upgrade to this new host endpoint.